### PR TITLE
API-2650: Use route hooks for ApiPage

### PR DESCRIPTION
**Pull request by anthony-kopczyk**
### Description
Updates the ApiPage component to use route hooks

### Process
- [N/A] Tests added or updated for change
- [N/A] Docs updated
- [N/A] Accessibility concerns have been considered and addressed

### Requested feedback

Not requested feedback but a note.

This component uses the location hook only to inject `location` to a child component. That child component is from swagger UI, and we are not converting those to use hooks at the moment. At the moment, that work isn't planned to be done so we''ll need to continue the use of location in this component for now.

### CODE REVIEW

**Review by TS000 — APPROVED** [review]
Looks great! Love how much these hooks are cleaning things up!

**Review by mjlumetta — APPROVED** [review]

**Review by johanna-adhoc — APPROVED** [review]
💯

File: src/containers/documentation/ApiPage.tsx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import classNames from 'classnames';
-import { Flag } from 'flag';
 import { useLocation, useParams } from 'react-router-dom';
-
+import * as React from 'react';
```
Comment (TS000) [inline src/containers/documentation/ApiPage.tsx:3]: I really like having the react import at the top and in then in this case following by the router hooks

Comment (anthony-kopczyk) [reply src/containers/documentation/ApiPage.tsx]: Went ahead and moved this this morning.

**Review by TS000 — APPROVED** [review]

**Pull request merged**